managed-ledger/src/test/java/org/apache/bookkeeper/mledger/impl/ManagedCursorInfoMetadataTest.java

```diff
@@ -73,8 +72,8 @@ public void testEncodeAndDecode(String compressionType) throws IOException {
             IllegalArgumentException compressionTypeEx = expectThrows(IllegalArgumentException.class, () -> {
                 new MetaStoreImpl(null, null, null, compressionType);
             });
-            assertEquals("No enum constant org.apache.bookkeeper.mledger.proto.MLDataFormats.CompressionType."
-                    + compressionType, compressionTypeEx.getMessage());
+            assertEquals(compressionType, compressionTypeEx.getMessage(),
```

Comment:
the original code was checking the error message, you have to keep two parameters here

Reply:
Good catch!